src/main/resources/templates/categories/space-heaters/spaceHeaterPackagesBoilerCalculator.ftl

```diff
@@ -0,0 +1,16 @@
+<#include '../../layout.ftl'>
+<#import 'spaceHeaterPackagesCalculatorCommon.ftl' as commonFields>
+
+<@common.standardProductForm
+title="Packages of space heater, temperature control and solar device energy label calculator: ${preferentialHeater}"
+showInsetText=false
+>
+  <@commonFields.commonSpaceHeaterPackagesCalculatorFields/>
+
+  <@govukFieldset.fieldset legendHeading="Supplementary heat pump" legendSize="h2">
```

Comment:
I think we can get rid of these fieldsets. Each set of questions is wrapped by a 'Does this package have...' question anyway, so the fieldset isn't wrapping any non-conditional questions

Reply:
done